### PR TITLE
Add aud registered claim to v2 context

### PR DESCRIPTION
**Pull request by OR13**
```json
{
  "@context": {
      "aud": {
      "@id": "https://www.iana.org/assignments/jwt#aud",
      "@type": "@id"
    }
  },
  "@id": "urn:uuid:123",
  "aud": ["urn:uuid:456", "https://api.example/verifier"]
}
```

```
<urn:uuid:123> <https://www.iana.org/assignments/jwt#aud> <https://api.example/verifier> .
<urn:uuid:123> <https://www.iana.org/assignments/jwt#aud> <urn:uuid:456> .

```

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1223.html" title="Last updated on Aug 4, 2023, 12:57 PM UTC (ca8de49)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1223/892eeac...ca8de49.html" title="Last updated on Aug 4, 2023, 12:57 PM UTC (ca8de49)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Review by dmitrizagidulin — APPROVED** [review]

**Review by msporny — CHANGES_REQUESTED** [review]
PR includes commits on main, needs a rebase.

File: contexts/credentials/v2 [hunk 1]
```diff
@@ -80,6 +84,10 @@
         "termsOfUse": {
           "@id": "https://www.w3.org/2018/credentials#termsOfUse",
           "@type": "@id"
+        },
+        "confidenceMethod": {
+          "@id": "https://www.w3.org/2018/credentials#confidenceMethod",
+          "@type": "@id"
```
Comment (OR13) [inline contexts/credentials/v2:90]:
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -2230,7 +2230,7 @@ <h3>Data Schemas</h3>
             </p>
             <p>
 If multiple schemas are present, validity is determined according to the
-processing rules outlined by each associated <code>credentialSchema</code> 
+processing rules outlined by each associated <code>credentialSchema</code>
```
Comment (OR13) [inline index.html:2233]:
```suggestion
```

File: index.html [hunk 3]
```diff
@@ -3555,6 +3555,26 @@ <h3>Reserved Extension Points</h3>
...
+                <p class="issue" title="(AT RISK) Reservation depends on implementations">
+This property reservation might be deleted in favor of an existing section
+in the specification if at least one specification with two independent
+implementations are demonstrated by the end of the Candidate Recommendation
+Phase. If that does not occur, this reservation will remain, but the existing
+section in the specification will be removed.
+See <a href="https://w3c-ccg.github.io/confidence-method-spec/">Verifiable Credential Confidence Methods</a>.
+                </p>
+              </td>
+            </tr>
```
Comment (OR13) [inline index.html:3577]:
```suggestion
```

File: index.html [hunk 4]
```diff
@@ -3588,13 +3588,14 @@ <h3>Reserved Extension Points</h3>
             <tr>
               <td>`renderMethod`</td>
               <td>
-A property used for specifying how to render a credential into a visual,
+A property used for specifying one or more methods to render a credential into a visual,
```
Comment (OR13) [inline index.html:3591]:
```suggestion
A property used for specifying how to render a credential into a visual,
```

File: index.html [hunk 5]
```diff
@@ -3625,6 +3625,82 @@ <h3>Reserved Extension Points</h3>
...
+compatible with the W3C Verifiable Credentials ecosystem if it is a
+<a>conforming document</a> and it utilizes at least one securing mechanism, as
+described by their respective requirements in this specification. While some communities might call some digital
+credential formats that are not <a>conforming documents</a>
+"verifiable credentials", doing so does NOT make that digital credential
+compliant to this specification.
+        </p>
+
+      </section>
+
```
Comment (OR13) [inline index.html:3703]:
```suggestion
```

**Review by davidlehn — CHANGES_REQUESTED** [review]
Extraneous commits should be removed so only the single relevant commit remains.

**Comment by OR13** [comment]
@davidlehn see same comment on https://github.com/w3c/vc-data-model/pull/1219#issuecomment-1665731653

**Review by brentzundel — APPROVED** [review]

**Comment by brentzundel** [comment]
@msporny and @davidlehn since this PR is quite simple (and the changes made were all by a single author) I think it should be fine to squash and merge back down to a single commit.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Normative, multiple reviews, no changes requested, no objections, merging.

**Pull request merged**